src/components/modal/v2/parts/ContentWrapper.jsx

```diff
@@ -5,28 +5,33 @@ import { useContent, useServerData, useProductMeta } from '../lib';
 import Header from './Header';
 import Container from './Container';
 import Overlay from './Overlay';
-import { LongTerm, ShortTerm } from './views';
+import { LongTerm, ShortTerm, ProductList } from './views';
 
 const ContentWrapper = () => {
     const { views } = useServerData();
     const contentWrapper = useRef();
     const contentBackground = useRef();
     const contentBodyRef = useRef();
-    const [product, setProduct] = useState();
-    const content = useContent(product);
-    const productMeta = useProductMeta(product);
 
-    if (views?.length > 0) {
-        setProduct(views[0].meta.product);
+    let defaultProduct;
+    if (views?.length === 1) {
+        defaultProduct = views[0].meta.product;
+    }
+    if (views?.length > 1) {
+        defaultProduct = 'PRODUCT_LIST';
```

Comment:
```suggestion
    let defaultProduct;
    
    // Filter out PRODUCT_LIST. Used for certain checks to determine whether or not to show product list view.
    const sanitizeViews = views.filter(view => view?.meta?.product !== 'PRODUCT_LIST');

    if (sanitizeViews?.length === 1) {
        defaultProduct = sanitizeViews[0]?.meta?.product;
    } else if (sanitizeViews?.length > 1) {
        defaultProduct = 'PRODUCT_LIST';
    }
```

How about we try the above. We can sanitize the views to filter out the product list, then in the if else checks, we check whether or not to show just the product or product list views.